dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoJedisClient.java

```diff
@@ -533,6 +533,42 @@ public Long execute(final Jedis client, final ConnectionContext state) throws Dy
         });
     }
 
+    
+    @Override
+    public ScanResult<Map.Entry<String, String>> hscan(final String key, final int cursor) {
```

Comment:
I agree with you; IMHO locking the Dyno client to a specific version of the Jedis Client is not bad. The current path of Jedis and Redis seem to be to focus on the Redis Cluster. Hence, as long as Dyno provides extra features to the client, I do not see anybody switching to the vanilla Jedis client from Dyno. Once a new client API is introduced for single node Redis and that API call is widely requested, we can revisit this. Hence, shall we extend ScanResult instead of overriding it?